tests/InfillTest.cpp

```diff
@@ -100,7 +100,7 @@ class InfillTestParameters
         , result_polygons(std::move(result_polygons))
     {
         // FIXME: Once we are using spdlog as logger, we'll also use fmt::format() here, see CURA-8258.
-        name = makeName("InfillTestParameters_P%d_Z%d_C%d_L%lld__%lld", static_cast<int>(params.pattern), static_cast<int>(params.zig_zagify), static_cast<int>(params.connect_polygons), params.line_distance, test_polygon_id);
+        name = makeName("InfillTestParameters_P%d_Z%d_C%d_L%lld__%zu", static_cast<int>(params.pattern), static_cast<int>(params.zig_zagify), static_cast<int>(params.connect_polygons), params.line_distance, test_polygon_id);
```

Comment:
```suggestion
        name = makeName(fmt::format("InfillTestParameters_P{}_Z{}_C{}_L{}__{}", static_cast<int>(params.pattern), static_cast<int>(params.zig_zagify), static_cast<int>(params.connect_polygons), params.line_distance, test_polygon_id));
```
An untested suggestion which would require `#include <fmt/format.h>` and the additional removal of the now redudant TODO on L102, also need to double check if this requires specific formatters